debian/copyright

```diff
@@ -0,0 +1,72 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Contact: Christopher Rosell <chrippa@tanuki.se>
```

Comment:
This line still refers to the maintainer of Livestreamer and needs to be changed.